tests/test_api.rs

```diff
@@ -8673,3 +8673,71 @@ fn test_fast_calls_onebytestring() {
   eval(scope, source).unwrap();
   assert_eq!("fast", unsafe { WHO });
 }
+
+#[test]
+fn gc_callbacks() {
+  let _setup_guard = setup();
+
+  #[derive(Default)]
+  struct GCCallbackState {
+    mark_sweep_calls: u64,
+    incremental_marking_calls: u64,
+  }
+
+  extern "C" fn callback(
+    _isolate: *mut v8::Isolate,
+    r#type: v8::gc::GCType,
+    _flags: v8::gc::GCCallbackFlags,
```

Comment:
use `v8::GCType` there is no `v8::gc` namespace in the C++ API. They are already prefixed with "GC".